code/modules/atmospherics/machinery/components/binary_devices/valve.dm

```diff
@@ -42,10 +42,12 @@ It's like a regular ol' straight pipe, but you can turn it on and off.
 		var/datum/pipeline/parent1 = parents[1]
 		parent1.reconcile_air()
 		investigate_log("was opened by [usr ? key_name(usr) : "a remote signal"]", INVESTIGATE_ATMOS)
+		balloon_alert_to_viewers("valve opened")
```

Comment:
any reason these are to all viewers? most balloon alerts dont do this because it clutters the screen